RPGLootFeed/LootDisplay/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua

```diff
@@ -612,6 +612,7 @@ function LootDisplayRowMixin:StyleText()
 		if enabledSecondaryRowText and self.secondaryText ~= nil and self.secondaryText ~= "" then
 			self.SecondaryText:ClearAllPoints()
 			self.SecondaryText:SetJustifyH(anchor)
+			self.SecondaryText:SetTextColor(unpack(defaultColor))
```

Comment:
I've got something like this in the v1.26.0 PR where I just do it as part of the `Init` (along with PrimaryText since that just happens to have hex code colors at every point in the strings). So I will probably stick with that, but same end goal, so this will be finally be fixed one way or another!